src/server/src/main/java/io/cassandrareaper/resources/RepairRunResource.java

```diff
@@ -207,10 +210,11 @@ public Response addRepairRun(
               .nodes(nodesToRepair)
               .datacenters(datacentersToRepair)
               .blacklistedTables(blacklistedTableNames)
-              .repairThreadCount(repairThreadCountParam.orElse(context.config.getRepairThreadCount()));
-
-      final RepairUnit theRepairUnit = repairUnitService.getOrCreateRepairUnit(cluster, builder);
+              .repairThreadCount(repairThreadCountParam.orElse(context.config.getRepairThreadCount()))
+              .timeout(timeout);
 
+      // Repair unit overlaps shouldn't affect one shot repairs. Setting the "force" setting to true.
+      final RepairUnit theRepairUnit = repairUnitService.getOrCreateRepairUnit(cluster, builder, true);
```

Comment:
I'm not sure I agree with this. Maybe I need more context… 

It should not be possible to run a on-shot repair over the top of an existing repair unit, without first warning the user. The confirmation from the user then becomes the explicit `force=true`.

Reply:
`force` got parameterised 💪